app/serializers/gist-revision.js

```diff
@@ -0,0 +1,7 @@
+import GistSerializer from "./gist";
+
+export default GistSerializer.extend({
+  normalizeQueryResponse: function(store, primaryModelClass, payload, id, requestType) {
```

Comment:
I'm querying for a record using two parameters. Ember Data expects an array back when you query. So I wrap the payload in an array.